crates/bevy_render/src/base_bundle.rs

```diff
@@ -0,0 +1,59 @@
+use bevy_ecs::prelude::Bundle;
+use bevy_transform::prelude::{GlobalTransform, Transform};
+
+use crate::view::{ComputedVisibility, Visibility};
+
+/// A [`Bundle`] with the following [`Component`](bevy_ecs::component::Component)s:
+/// * [`Visibility`] and [`ComputedVisibility`], which describe the visibility of an entity
+/// * [`Transform`] and [`GlobalTransform`], which describe the position of an entity
+///
+/// * To show or hide an entity, you should set its [`Visibility`].
+/// * To get the computed visibility of an entity, you should get its [`ComputedVisibility`].
```

Comment:
Maybe note that it is only valid after the visibility systems have been run? (check_visibility and visibility_propagate, I think.)

Reply:
On the one hand, this is useful information. On the other hand, idk if we should be duplicating it across every bundle that includes ComputedVisibility. ComputedVisibility does have this "label dependency" information encoded in the relevant fields already. But it is probably worth also including that in the main ComputedVisibility docs.

Reply:
(Not something I think we should block on here though)

Reply:
https://github.com/bevyengine/bevy/issues/5372